src/main/kotlin/no/nav/tiltakspenger/skjerming/SkjermingService.kt

```diff
@@ -34,30 +36,77 @@ class SkjermingService(
         }.register(this)
     }
 
+    fun loggVedInngang(packet: JsonMessage) {
+        logg.info(
+            "løser behov med id {} og korrelasjonsid {}",
+            keyValue("id", packet["@id"].asText()),
+            keyValue("behovId", packet["@behovId"].asText())
+        )
+        sikkerlogg.info(
+            "løser behov med id {} og korrelasjonsid {}",
+            keyValue("id", packet["@id"].asText()),
+            keyValue("behovId", packet["@behovId"].asText())
+        )
+        sikkerlogg.debug { "mottok melding: ${packet.toJson()}" }
+        sikkerlogg.debug { "mottok melding: ${packet.toJson()}" }
+    }
+
+    private fun loggVedUtgang(packet: JsonMessage, løsning: () -> String) {
+        logg.info(
+            "har løst behov med id {} og korrelasjonsid {}",
+            keyValue("id", packet["@id"].asText()),
```

Comment:
Jeg er ikke helt sikker på hvordan disse keyValue (structured arguments) faktisk vil ende opp med å se ut i Kibana, så dette må nok testes ut litt. Jeg ser at Dagpenger bruker MDC for å få med id og behovId, ref https://github.com/navikt/dp-behovsakkumulator/blob/master/src/main/kotlin/no/nav/dagpenger/behovsakkumulator/Behovsakkumulator.kt
```
private fun loggBehov(packet: JsonMessage) {
        withLoggingContext(
            "behovId" to packet["@id"].asText(),
            "vedtakId" to packet["vedtakId"].asText()
        ) {
            listOf(log, sikkerLogg).forEach { logger ->
                logger.info {
                    val løsninger = packet["@løsning"].fieldNames().asSequence().joinToString(", ")

                    "Mottok løsning for $løsninger"
                }
            }
        }
    }
    ```